README.md

```diff
@@ -381,3 +381,8 @@ By participating, you are expected to honor this code.
 2. Hack.
 3. Push the branch back to GitHub.
 4. Send a pull request to our upstream repo.
+
+# Releasing
+``mvn -P release release:prepare``
```

Comment:
Can you add some comments about what these commands do

Reply:
@dmichel1 updated.

Reply:
tack!